posts/inside-rust/2020-07-24-opening-up-the-core-team-agenda.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Opening up the Core Team agenda"
+author: Pietro Albini
+team: the Core Team <https://www.rust-lang.org/governance/teams/core>
+---
+
+The Core Team tends to project-wide policy questions on all sorts of matters,
```

Comment:
Something in this wording is a bit off to me, but I'm not sure what. I think "tends" is the wrong verb, or maybe it's missing something, I'm not sure 😅

Reply:
It's a garden path sentence, it initially reads as the core team tending towards a behavior rather than the core team taking care of something

Reply:
```suggestion
The Core Team works on project-wide policy questions on all sorts of matters,
```

perhaps?